nx/lib/nx.ex

```diff
@@ -9167,6 +9167,84 @@ defmodule Nx do
     end
   end
 
+  @doc """
+  Finds the variance of a tensor.
+
+  ## Examples
+
+  iex> Nx.var(Nx.tensor([[1, 2], [3, 4]]))
+
+  #Nx.Tensor<
+  f32
+  1.25
+  >
+  """
+  @doc type: :element
+  @spec var(tensor :: Nx.Tensor.t()) :: Nx.Tensor.t()
+  def var(%Nx.Tensor{shape: shape} = tensor) do
+    total =
+      shape
+      |> Tuple.to_list()
+      |> Enum.reduce(fn val, acc -> acc * val end)
+
+    mean = mean(tensor)
+
+    tensor
+    |> subtract(mean)
+    |> power(2)
+    |> sum()
+    |> divide(total)
+  end
+
+  def var(_), do: raise(ArgumentError, "must be a tensor")
+
+  @doc """
+  Finds the standard deviation of a tensor.
+
+  ## Examples
+
+  iex> Nx.std(Nx.tensor([[1, 2], [3, 4]]))
+
+  #Nx.Tensor<
+  f32
+  1.1180340051651
+  >
+  """
+  @doc type: :element
+  @spec std(tensor :: Nx.Tensor.t()) :: Nx.Tensor.t()
+  def std(%Nx.Tensor{} = tensor) do
+    sqrt(var(tensor))
+  end
+
+  def std(_), do: raise(ArgumentError, "must be a tensor")
+
+  @doc """
+  Normalizes the tensor by using standard scale.
+
+  ## Examples
+
+  iex> Nx.normalize(Nx.tensor([[1, 2], [3, 4]]))
+
+  #Nx.Tensor<
+  f32[2][2]
+  [
+    [-1.3416407108306885, -0.4472135901451111],
+    [0.4472135901451111, 1.3416407108306885]
+  ]
+  >
+  """
+  @doc type: :element
+  @spec normalize(tensor :: Nx.Tensor.t()) :: Nx.Tensor.t()
+  def normalize(%Nx.Tensor{} = tensor) do
```

Comment:
I think we should either move this to another module (such as `Nx.Statistics`) or rename this function, because without a specific context, one could expect normalize to be `def normalize(t), do: Nx.divide(t, Nx.LinAlg.norm(t))`

Reply:
For all of these functions, we could also have a "per axis" version, like the ones we have for `sum`.
Especially in the case where we're dealing with a set of random variables, this could be really useful.

WDYT of having a follow-up PR for this? If you can't tackle this (lack of time, etc), let us know and we can add it to the issues tracker.

Reply:
I can definitely add a follow-up PR next week to address this.